homeassistant/components/zwave_js/__init__.py

```diff
@@ -274,6 +293,47 @@ def async_on_notification(
 
         hass.bus.async_fire(ZWAVE_JS_NOTIFICATION_EVENT, event_data)
 
+    @callback
+    def async_on_value_updated(
+        value_updates_disc_info: list[ZwaveDiscoveryInfo], value: Value
+    ) -> None:
+        """Fire value updated event."""
+        # Get the discovery info for the value that was updated.
+        try:
+            disc_info = next(
+                disc_info
+                for disc_info in value_updates_disc_info
+                if disc_info.primary_value.value_id == value.value_id
+            )
+        except StopIteration:
+            return
+
+        device = dev_reg.async_get_device({get_device_id(client, value.node)})
+        event_data = {
+            ATTR_DOMAIN: DOMAIN,
```

Comment:
Is this needed? All of our custom events will be the same domain.

Reply:
No I copied it from our other events but none of them need it

Reply:
Hmm, is that some standard that I've forgotten about? It does seem redundant.

Reply:
No, I think it was added to the `value notification` event and we missed it, then I copied it into the `notification` event for consistency. I don't see any value in it, but theoretically it would be a breaking change to remove it